app/controllers/redirections_controller.rb

```diff
@@ -100,7 +101,7 @@ def ensure_request_is_not_from_a_bot
   def ensure_request_is_not_a_subdomain_of_an_existing_site
     if referrer
       referrer_without_leading_subdomain = URI.parse(referrer).host.
-        sub(/^[^.]+\./, '')
+        sub(/^[^.]+\./, "")
```

Comment:
nice.

Reply:
All credit to rufo 😄 https://github.com/ruby-formatter/rufo